tokencost/costs.py

```diff
@@ -50,7 +50,7 @@ def count_message_tokens(messages: List[Dict[str, str]], model: str) -> int:
         "gpt-4-32k-0314",
         "gpt-4-0613",
         "gpt-4-32k-0613",
-    }:
+    } or model.startswith('ollama/'):
```

Comment:
I'm not sure this is how Ollama actually counts messages. Worth adding a TODO comments here and an extra issue to account for models with non-standard per-message  token additions.

Reply:
In that case, we can close this PR and create a new issue.